tests/blackbox/logger/redact.test.ts

```diff
@@ -5,9 +5,10 @@ import { TestLogger } from '@common/test-logger';
 import { awaitDirectusConnection } from '@utils/await-connection';
 import { ChildProcess, spawn } from 'child_process';
 import { EnumType } from 'json-to-graphql-query';
-import knex from 'knex';
 import type { Knex } from 'knex';
+import knex from 'knex';
 import { cloneDeep } from 'lodash';
+import path from 'path';
```

Comment:
```suggestion
import path from 'node:path';
```

understandably there's still quite a number of places where we aren't using `node:` prefix when importing built in modules, but maybe we should start doing so going forward if that's okay (and maybe update other files with node imports in a another ticket/PR) 👍

We can also leave this for now to do it altogether afterwards!